frameworks/g-cloud-9/questions/services/emailOrTicketingSupport.yml

```diff
@@ -19,10 +19,13 @@ type: radios
 options:
   - label: "Yes"
     value: "yes"
+    filter_label: "Email/online ticketing"
```

Comment:
Generally we only use quotes around YAML words like yes/no/on/off which are otherwise interpreted as booleans. Longer strings aren't quoted in many other places.